src/sorcha/modules/PPCalculateApparentMagnitudeInFilter.py

```diff
@@ -8,7 +8,7 @@
 
 
 def PPCalculateApparentMagnitudeInFilter(
-    padain, function, colname="TrailedSourceMag", lightcurve_choice="None"
+    padain, function, colname="TrailedSourceMag", lightcurve_choice=None
```

Comment:
The default value seemed to bounce around a bit between where the config value was defined in the .ini file and where it finally is used. 

In PPConfigParser, if the .ini file defines this parameter as the string `"none"` it will be cast to the Python `None`. But then here we were defining the default value to be the string `"none"` again. So I've updated the default value to be `None` to be in alignment with what `PPConfigParser` will output.